python/torch_mlir_e2e_test/test_suite/elementwise.py

```diff
@@ -413,6 +413,28 @@ def ElementwiseLeakyReluModule_basic(module, tu: TestUtils):
 # ==============================================================================
 
 
+class ElementwisePreluModule(torch.nn.Module):
+
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+        ([2, 3, 4], torch.float32, True),
+        ([2, 3, 4], torch.float32, True),
+    ])
+    def forward(self, x, weight = torch.tensor([0.25])):
```

Comment:
Can you pass the `weight` tensor as an argument to `forward` to be consistent with the style in this file?

Reply:
Yes, After doing that, there is 
 error: unsupported by backend contract: tensor with unknown rank 
is coming.